config/environment.js

```diff
@@ -19,6 +19,8 @@ module.exports = function (environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
+      municipalities: 'gemeentes',
+      provinces: 'provincies',
```

Comment:
I felt it was more readable to do it this way as I really dislike seeing Dutch words used as hard-coded strings

Reply:
I agree, having "Dutch words" spread in the source code is something that bother me for a while. 

But I'm not sure that using environment variables is the best option. If we move all Dutch there it will turn messy. Moreover, ENV variables are primarily made to configure application from the outside, these queryParams are not plan to be different based on environments.  

I would go into something more simple like a constants folder: `constants/query-params.ts`

Reply:
On it!